src/lib.rs

```diff
@@ -67,27 +67,36 @@ pub type Result<T> = ::std::result::Result<T, Error>;
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 struct FrugalosConfigWrapper {
     #[serde(rename = "frugalos")]
+    #[serde(default)]
     config: FrugalosConfig,
 }
 
 /// frugalos の設定を表す struct。
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct FrugalosConfig {
     /// データ用ディレクトリのパス。
+    #[serde(default)]
     pub data_dir: String,
     /// ログをファイルに出力する場合の出力先ファイルパス。
+    #[serde(default)]
     pub log_file: Option<PathBuf>,
     /// 出力するログレベルの下限。
+    #[serde(default)]
```

Comment:
It seems better to replace this attribute by `#[serde(default = "default_loglevel")]` for being consistent with the implementation of `Default` trait.

Reply:
Ok. Thank you for reviewing.

Reply:
Although I approved this PR, I think that `[serde(default)]` (the original code) is preferable to `#[serde(default = "default_loglevel")` (the new one) with respect to the point to be consistent with the implementation of Default trait because `[serde(default)]` uses `std::default::Default` of `sloggers::types::Severity`.

Can you explain me why `#[serde(default = "default_loglevel")]` is preferable to `[serde(default)]` ??